src/envs/blocks.py

```diff
@@ -23,25 +23,36 @@ class BlocksEnv(BaseEnv):
     """Blocks domain."""
     # Parameters that aren't important enough to need to clog up settings.py
     table_height = 0.2
-    block_size = 0.1
-    x_lb = 1.3
-    x_ub = 1.4
-    y_lb = 0.15
-    y_ub = 20.85
+    block_size = 0.045
+    # The table x bounds are (1.1, 1.6), but the workspace is smaller.
+    # Make it narrow enough that blocks can be only horizontally arranged.
+    x_lb = 1.35 - block_size
+    x_ub = 1.35 + block_size
+    # The table y bounds are (0.3, 1.2), but the workspace is smaller.
+    y_lb = 0.4
+    y_ub = 1.1
+    pick_z = 0.75
+    robot_init_x = (x_lb + x_ub) / 2
+    robot_init_y = (y_lb + y_ub) / 2
+    robot_init_z = pick_z
     held_tol = 0.5
-    open_fingers = 0.8
-    pick_tol = 0.08
+    open_fingers = 0.04
+    closed_fingers = 0.01
+    finger_tol = 0.00001
+    pick_tol = 0.0001
+    on_tol = 0.01
```

Comment:
why does on_tol need to be higher?

Reply:
the blocks can be slightly off center from each other after a stack.